examples/Wiper/Wiper.ino

```diff
@@ -0,0 +1,49 @@
+/*
+  Using a servo as a car Wiper, activated by switch.
+
+  The circuit:
+  - switch is attached from pin 2 to ground
+  - Servo's signal pin is attached to Pin 9
+
+  modified on 21 March 2020
+  by Durgesh Pachghare
+*/
+
+#include <Servo.h>
+
+Servo myservo;  // create servo object to control a servo
+
+int button_pin = 2;  // digital pin used to connect the button input
+int val;    // variable to read the value from the analog pin
+int startpos = 10;      //angle from which the servo wiper starts
+int endpos = 145;       //angle till the servo wiper rotates
+
+void setup()
+{
+  myservo.attach(9);  // attaches the servo on pin 9 to the servo object
+  pinMode(button_pin, INPUT_PULLUP);   //declares the pin as digital input
+  myservo.write(startpos);      // Initialize the servo wiper to start angle
+  pinMode(LED_BUILTIN, OUTPUT); // to show the state of servo wiper
+}
+
+void loop()
+{
+  //The button_pin is declared as INPUT_PULLUP which means switch logic is inverted.
+  //It gives LOW when the switch is pressed and HIGH when it is open
+  if (digitalRead(button_pin) == LOW)  //Start the servo if the button is pressed and complete the
+    //entire rotation regardless of button state later

```

Comment:
```suggestion
                                       //entire rotation regardless of button state later
```
I think this makes it more clear that the comment is a continuation of the previous line.

Reply:
Yes. Thank you so much. I will follow this now onwards. :)